.travis.yml

```diff
@@ -1,9 +1,5 @@
-sudo: required
```

Comment:
Why remove `sudo: required?` I see that it should provide faster boot times but less memory (which I guess isn't an issue).
https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments